Sources/SwiftMemcache/MemcachedResponse.swift

```diff
@@ -0,0 +1,23 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the swift-memcache-gsoc open source project
+//
+// Copyright (c) 2023 Apple Inc. and the swift-memcache-gsoc project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of swift-memcache-gsoc project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import NIOCore
+
+enum MemcachedResponse {
```

Comment:
I don't know if modelling this as an enum is correct. From the return codes we can't know what command was issued e.g. `HD` and `EN` is used by both get and set commands. Let's just model this out as a struct that has a `returnCode`. We are going to do the request-response matching in the handler in the end.